src/libs/actions/Report.ts

```diff
@@ -998,7 +998,12 @@ function markCommentAsUnread(reportID: string, reportActionCreated: string) {
 
     // Find the latest report actions from other users
     const latestReportActionFromOtherUsers = Object.values(reportActions ?? {}).reduce((latest: ReportAction | null, current: ReportAction) => {
-        if (current.actorAccountID !== currentUserAccountID && (!latest || current.created > latest.created)) {
+        if (
+            current.actorAccountID !== currentUserAccountID &&
```

Comment:
The style is odd, this should go after the if not in a new line 🤔

Reply:
Prettier doesn't allow me to move it after the if

Reply:
There are a few codes that has the same style, for example,
https://github.com/Expensify/App/blob/ed7029b33c5711736569aea8981e21fc68e12d89/src/pages/home/report/ReportActionsList.tsx#L219-L225